### ISSUE TITLE
Possible to collect everything?

### ISSUE THREAD

**Issue by richardpenner** [assistunion/xml-stream#37]
This is a really great library. I'm parsing an XML file of 300MB with no issues. Just a question though: is there a way to collect everything? The XML I'm parsing is generated by someone else, and I'd prefer collection to happen by default, such that I don't have to pre-determine each element that could possibly have multiple entries.

**Comment by michelem09** [comment]
+1 I think this should be a standard for this library. Collection must be on everything, I cannot know where there are array of elements.

**Comment by dimatter** [comment]
+1 !!!!!

**Comment by TimoRuetten** [comment]
Yes ... its sad that this does not work. +1 also from me - we will remove this dependency because its not useful if you have multiple elements with the same tag in the children you are collecting.
